modules/xmpp/recording.js

```diff
@@ -61,6 +61,7 @@ Recording.status = {
     UNAVAILABLE: 'unavailable',
     PENDING: 'pending',
     RETRYING: 'retrying',
+    ERROR: 'error',
```

Comment:
It's not used in this file but jitsi-meet uses it for one of its cases with Recording.js. jitsi-meet js had these constants, plus error, in Recording.js with a todo to use the constants in lib.